source/gui/startupDialogs.py

```diff
@@ -153,9 +153,17 @@ def __init__(self, parent):
 		groupLabel = _("License Agreement")
 		sizer = wx.StaticBoxSizer(wx.VERTICAL, self, label=groupLabel)
 		sHelper.addItem(sizer)
-		licenseTextCtrl = wx.TextCtrl(self, size=(500, 400), style=wx.TE_MULTILINE | wx.TE_READONLY | wx.TE_RICH)
+		licenseTextCtrl = wx.TextCtrl(
+			self,
+			size=(-1, 500),  # -1 uses default width
```

Comment:
The 500 value will appear differently based on the windows scaling. We can use `DpiScalingHelperMixinWithoutInit` to to scale that value so it appears the same.